src/screens/runPlan/runPlanList/PlanItemListItem.tsx

```diff
@@ -49,7 +49,9 @@ export class PlanItemListItem extends React.PureComponent<Props> {
 
   handlePress = () => {
     if (this.props.item.type === PlanItemType.ComplexTask && this.isItemParentPlan()) {
-      return this.navigateToRunPlanSubItemsList;
+      if(this.props.item.completed === false) {
```

Comment:
Can't it be single if/else with below statement?
```
this.props.item.type === PlanItemType.ComplexTask && this.isItemParentPlan() && !this.props.item.completed
```

If not, we should provide some value for case when `this.props.item.completed` equals `true`.